src/user/proposals/AdminProposalDashboard/DashboardContent/DashboardContent.js

```diff
@@ -25,150 +157,118 @@ class DashboardContent extends Component {
-                <div className="proposal-details">
-                  ex sit ex laboris adipisicing enim eiusmod proident
-                  exercitation ea fugiat in llit pariatur occaecat ut nostrud
-                  ullamco ex official ex sit ex laboris adipisicing enim eiusmod
-                  proident exercitation ea...
-                </div>
-                <div className="proposal-options">
-                  <Button variant="primary" className="option-btn" size="sm">
-                    <span className="option-text">Edit</span>
-                  </Button>
-                  <Button
-                    variant="primary"
-                    className="option-btn"
-                    size="sm"
-                    active
-                  >
-                    <span className="option-text">View</span>
-                  </Button>
-                </div>
-              </div>
-            </div>
-            <div className="single-proposal">
-              <div className="user-info">
-                <div className="image">
-                  <Image src={userIcon2} alt="icon" rounded />
-                </div>
-                <div className="img-desc">
-                  <h2>Admin panel in donut dashboard</h2>
-                  <p className="proposal-date">June 5, 2018 4:31 AM</p>
-                </div>
                 <Button
                   variant="primary"
-                  className="status-btn-accepted"
-                  size="sm"
+                  className="category-btn"
+                  onClick={() => this.handleButtonClick("Rejected")}
                 >
-                  <span className="btn-content">Accepted</span>
+                  <span className="btn-content">Rejected</span>
                 </Button>
               </div>
-              <div className="proposal-content">
-                <div className="proposal-details">
-                  ex sit ex laboris adipisicing enim eiusmod proident
-                  exercitation ea fugiat in llit pariatur occaecat ut nostrud
-                  ullamco ex official ex sit ex laboris adipisicing enim eiusmod
-                  proident exercitation ea...
-                </div>
-                <div className="proposal-options">
-                  <Button variant="primary" className="option-btn" size="sm">
-                    <span className="option-text">Edit</span>
-                  </Button>
-                  <Button
-                    variant="primary"
-                    className="option-btn"
-                    size="sm"
-                    active
-                  >
-                    <span className="option-text">View</span>
-                  </Button>
-                </div>
-              </div>
             </div>
           </div>
         </div>
+
+        <div className="proposal-container">
+          <div className="proposals">
+            {this.state.displayItems.map((proposalItem, index) => {
```

Comment:
This will throw error when this.state.displayItems will be empty [], try once. Handle it